Kitodo/src/main/java/de/sub/goobi/forms/HelperForm.java

```diff
@@ -226,30 +126,6 @@ public String getServletPathWithHostAsUrl() {
         return list;
     }
 
-    /**
-     * Method returns a valid css file, which is the suggestion unless suggestion is
-     * not available if not available default.css is returned.
-     *
-     * @param cssFileName
-     *            suggested css file
-     *
-     * @return valid css file
-     */
-    public String getCssLinkIfExists(String cssFileName) {
```

Comment:
Is this whole CSS theming function in some kind of usage? If not, then is a cleanup in a separate pull request needed.